packages/paste-core/core-bundle/src/datagrid.tsx

```diff
@@ -0,0 +1 @@
+export * from '@twilio-paste/datagrid';
```

Comment:
Did the package name change by any chance? I think this one needs deleting

Reply:
It did, great catch!